components/camel-git/src/main/docs/git-component.adoc

```diff
@@ -83,5 +83,17 @@ from("git:///tmp/testRepo?type=commit")
     .to(....)
 ---------------------------------------
 
+== Custom config file
+By default camel-git will load .gitconfig file from user home folder. You
```

Comment:
Use quotes to force the formatting to recognize this as a file. For instance, to make it like this `.gitconfig`.

Reply:
Done.